hack/make/build-rpm

```diff
@@ -135,8 +135,14 @@ set -e
 		# selinux policy referencing systemd things won't work on non-systemd versions
 		# of centos or rhel, which we don't support anyways
 		if [ "${suite%.*}" -gt 6 ] && [[ "$version" != opensuse* ]]; then
+            selinuxDir="selinux"
+            case "$version" in
+                fedora-24)
+                    selinuxDir="selinux-fedora"
```

Comment:
can the folder just be called `selinux-fedora-24` that way you can just test if it exists and replace the default value with it in that case ? (and it'll be faster to add new exceptions)